Capfile

```diff
@@ -0,0 +1,13 @@
+require "capistrano/setup"
+require "capistrano/deploy"
+
+require "capistrano/scm/git"
+install_plugin Capistrano::SCM::Git
+
+require "capistrano/rbenv"
+require "capistrano/bundler"
```

Comment:
Let's just require `capistrano/rails` instead of breaking it out into `bundler`, `rails/assets`, etc.